src/main/java/frc/robot/commands/auto/Auto.java

```diff
@@ -17,17 +20,18 @@ public class Auto extends SequentialCommandGroup {
 
 
     public Auto(ArmSubsystem arm, DriveSubsystem drive, double driveTime) {
+        addRequirements(arm);
+        addRequirements(drive);
 
         addCommands(
-            new InstantCommand(() -> arm.setMode(Mode.SHOOT_HIGH)),
-            new WaitCommand(1),
-            new InstantCommand(() -> arm.setMode(Mode.IDLE)),
-            new ParallelRaceGroup(
+            new ArmMove(arm, Mode.SHOOT_HIGH).raceWith(new WaitCommand(1.0)),
+            new ParallelRaceGroup (
```

Comment:
Is ParallelRaceGroup correct here? Will ArmMove complete and end the race?

Reply:
ArmMove never is finished, so it shouldn't win the race against the wait